keyboards/scrabblepad/keymaps/random/keymap.c

```diff
@@ -0,0 +1,75 @@
+#include "scrabblepad.h"
+#include <stdlib.h>
+
+static uint16_t keystroke;
+static bool shift_pressed;
+
+static int lower = (int) KC_A;
+static int upper = (int) KC_Z;
+
+uint16_t random_keycode(void) {
+  return lower + (rand() % (upper - lower + 1));
+};
+
+enum custom_keycodes {
+  RND_KEY = SAFE_RANGE
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch(keycode) {
+    case RND_KEY:
+      if (record->event.pressed) {
+        keystroke = random_keycode();
+        if (rand() % 2 > 0) {
+          shift_pressed = 1;
+          register_code(KC_LSHIFT);
+        }
+        register_code(keystroke);
+      } else {
+        if (shift_pressed > 0) {
+          unregister_code(KC_LSHIFT);
+        }
+        unregister_code(keystroke);
+      }
+
+      return false;
+
+    default:
+        return true;
+  }
+
+  return true;
```

Comment:
This is unnecessary since the `switch` returns `true` by default. Removing it will also force you to terminate all cases properly, with a `return`.
```suggestion
```